### PR TITLE
Fix UnsignedInteger test module

### PR DESCRIPTION
**Pull request by GianfrancoBazzani**
# Fix UnsignedInteger test module

## Description

Fixed crate imports as https://github.com/lambdaclass/lambdaworks/pull/75 for both U384 and U256 types.
Added Byte conversion test for U256 type.

## Checklist

- [x] Unit tests added.

### CODE REVIEW

File: math/src/unsigned_integer/element.rs [hunk 1]
```diff
@@ -1755,4 +1739,32 @@ mod tests_u256 {
...
+        assert_eq!(number.to_bytes_le(), expected_bytes);
+    }
+
+    #[test]
+    fn from_bytes_be_works() {
+        let bytes = vec![0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1];
+        let expected_number = U256::from_u64(1);
+        assert_eq!(U256::from_bytes_be(&bytes).unwrap(), expected_number);
+    }    #[test]
+    fn from_bytes_le_works() {
```
Comment (ilitteri) [inline math/src/unsigned_integer/element.rs:1765]: A couple of new lines are missing here

**Review by ilitteri — CHANGES_REQUESTED** [review]
Looks good! With the merge conflicts solved and the new lines added we could merge.

**Review by ilitteri — APPROVED** [review]
LGTM, great work!

**Comment by MauroToscano** [comment]
Looks good! Before merging we should check the CI is working. Can you merge with main ?

**Review by jrchatruc — APPROVED** [review]

**Review by ilitteri — CHANGES_REQUESTED** [review]
Could you merge with main so we could run the CI?

**Review by ilitteri — APPROVED** [review]

**Review by MauroToscano — APPROVED** [review]

**Pull request merged**